src/mrms/mrms_grib2ioda.py

```diff
@@ -60,7 +60,8 @@
 obsErrName = iconv.OerrName()
 qcName = iconv.OqcName()
 
-float_missing_value = -999.0    # or netCDF value,  nc.default_fillvals['f4']
+grib_missing_value = -999.0
```

Comment:
It seems to be a "missing value" for this specific dataset. Perhaps rename it to `mrms_grib_missing_value`?!